src/modules/claims/claims.service.ts

```diff
@@ -216,6 +218,38 @@ export class ClaimsService {
         return this._cacheClient.issueClaim(this._signerService.did, message);
     }
 
+    /**
+     * @description Registers issued onchain claim with Claim manager
+     *
+     * @param claimId - id of signed onchain claim
+     */
+    async registerOnchain(claimId: Claim["id"]) {
+        const claim = await this.getClaimById(claimId);
+        if (!isIssuedOnchain(claim)) {
+            throw new Error(ERROR_MESSAGES.CLAIM_WAS_NOT_ISSUED);
+        }
+        const { token, subjectAgreement, onChainProof } = claim;
+        const { claimData, sub } = this._didRegistry.jwt.decode(token) as {
+            claimData: { claimType: string; claimTypeVersion: number; expiry: number };
```

Comment:
`expiry` is not needed
```suggestion
            claimData: { claimType: string; claimTypeVersion: number };
```